converter/include/k4SimDelphes/DelphesEDM4HepOutputConfiguration.h

```diff
@@ -165,11 +165,11 @@ OutputSettings getEDM4hepOutputSettings(ExRootConfReader* confReader)
 
   settings.MissingETCollections = toVecString(
     confReader->GetParam("EDM4HepOutput::MissingETCollections"),
-    {"MissingET"});
+    {""});
 
    settings.ScalarHTCollections = toVecString(
     confReader->GetParam("EDM4HepOutput::ScalarHTCollections"),
-    {"ScalarHT"});
+    {""});
```

Comment:
I'm not sure you need to change the converter, maybe by accident?

Reply:
this was a local change indeed that I reverted. If we don't remove the hardcoded names in the converter, there is not the possibility to not save a collection by removing it from the output_config.tcl

Reply:
I think these are fixing a different kind of "problem". Without these changes, the converter will always fill these collections, even if you remove them from your output config file. I.e. the defaults here might override your configuration. Without these fixes you have to explicitly touch the delphes card and remove the branches you do not want from the `TreeWriter` config section. With these changes you can leave the delphes card untouched and simply suppress these outputs by removing them from the output configuration.

Reply:
BTW @tmadlener , using the default configuration for the output I do not see ```ScalarHT``` in the output root files.
I never noticed it because I am not using it until now while I was testing the collection removal

Reply:
That might be because `ScalarHT` is not in the `TreeWriter` configuration? By default the converter will only try to convert what is actually there.